dist/test_data.py

```diff
@@ -76,7 +80,11 @@ def __ge__(self, other):
     Config('update_threads', 0, r'''
         The number of threads performing update operations''', min=0, max=20),
     Config('update_config',0,r''',
-        The definition of the record being updated''', subconfig=load_config)
+        The definition of the record being updated''', subconfig=load_config),
+    Config('min_operation_per_transaction', 1, r'''
```

Comment:
Can we get a transaction subconfig our top level configuration is getting very polluted.